scapy/autorun.py

```diff
@@ -7,7 +7,7 @@
 Run commands when the Scapy interpreter starts.
```

Comment:
I will remove that commit once https://github.com/secdev/scapy/pull/631 has been pushed.